docs/get-started/sign-up.md

```diff
@@ -62,7 +57,41 @@ After 30 days, your Sumo Logic **Trial account** will convert to a **Free accoun
 <sup>3</sup> Forwards your logs to an S3 bucket after being collected and analyzed in Sumo Logic.<br/>
 <sup>4</sup> Notifies you in real-time when errors occur on your systems.
 
-For more information, see [Sumo Logic Account Usage](/docs/manage/manage-subscription).
+ To unlock premium features and scale your Sumo Logic usage as your needs grow, upgrade to a paid plan. For more information, see [Sumo Logic Pricing](https://www.sumologic.com/pricing) and [Manage Subscription](/docs/manage/manage-subscription).
+
+### Upgrade to a Flex plan
+
+Using our service checkout, you can quickly and easily sign up for our Flex Essentials plan with a credit card in just minutes.  
+
+The Flex Essentials plan is ideal for:  
+- Small- to medium-sized businesses seeking quick and independent onboarding.  
+- IT professionals, developers, and analysts who need rapid access to advanced features for security, monitoring, and analytics.  
+
+If your requirements exceed the capabilities of the Flex Essentials plan, you’ll need to [upgrade to an enterprise plan](#upgrade-to-an-enterprise-plan).
+
+#### How to use self-service checkout  
+
+1. Log in to your Sumo Logic account.  
+1. [**Classic UI**](/docs/get-started/sumo-logic-ui-classic). From the left nav menu, go to **Administration** > **Account** > **Manage Plan**. <br/>[**New UI**](/docs/get-started/sumo-logic-ui). In the top Sumo Logic nav menu, go to **Administration** > **Manage Plan**.
+1. Here, you can review the available Sumo Logic plans and pricing. Select the **Flex Essentials** plan.
+1. Enter your credit card details in the secure checkout form.  
+1. Review and confirm your purchase.  
+1. Once your payment has been processed, your account will be upgraded, and you’ll have immediate access to premium features.  
+
+For information on updating your payment information, see [Manage Billing Information](/docs/manage/manage-subscription/manage-billing-information).
+
+:::note limitations
+At this time:  
+- Customers using order forms for billing are not eligible to switch to self-service payments.  
+- Downgrades to the free plan cannot be completed through the checkout interface.  
+- Downgrades between paid plans take effect at the start of the next billing cycle.  
+:::
+
+If you encounter any issues during the upgrade process, contact our [support team](https://support.sumologic.com/support/s/).  
+
+### Upgrade to an enterprise plan
+
+Contact our [Sales Team](https://www.sumologic.com/contact-us/) for assistance. We'll consult with you to determine the best solution for your organization.
```

Comment:
I don't see any of the new plans mentioned. see picture below. 
![Screenshot 2024-11-25 at 9 15 20 AM](https://github.com/user-attachments/assets/9d31dc13-d5ba-4464-973d-f120ffa9d7eb)

Reply:
Thank you! Please see the updated doc [here](https://github.com/SumoLogic/sumologic-documentation/pull/4778/files).